### PR TITLE
Allow window override in dropdown auto-flip

### PR DESCRIPTION
**Pull request by interactivellama**
This allows explicitly setting auto-flip container to window via `data-target=window`.

### CODE REVIEW

**Comment by swilliamset** [comment]
Reconsider with less condition nesting

**Comment by interactivellama** [comment]
Updated with flatter decision tree @futuremint @swilliamset

File: js/dropdown-autoflip.js [hunk 1]
```diff
@@ -97,26 +97,27 @@
...
-			$.each(element.parents(), function (index, value) {
-				if ($(value).css('overflow') !== 'visible') {
-					containerElement = value;
+		// parse decision tree
+		var targetSelector = element.attr('data-target') || false;
+		var nonWindowTargetChosen = targetSelector !== 'window' && targetSelector;
+		var noTargetChosen = targetSelector !== 'window' && !targetSelector;
+
+		if (nonWindowTargetChosen) {
+			targetedElement = $(targetSelector);
```
Comment (futuremint) [inline js/dropdown-autoflip.js]: Is this an accidental global?

Comment (kevinparkerson) [inline js/dropdown-autoflip.js]: I was just going to comment on this... @interactivellama, when I run `grunt test` jshint catches this and says it is undefined

**Pull request merged**